app/assets/index.hbs

```diff
@@ -1,48 +1,237 @@
-    <link rel="icon" type="image/x-icon" href="favicon.ico">
-    <style>
-        body {
-            background-color: #2a2a2a;
-        }
-        .centerDiv {
-            width:140px; border-radius: 5px; color:lightgrey;
-            padding:10px; height:50px; position:absolute;
-            margin-top:-25px; margin-left:-70px; top:50%; left:50%;
-            font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
-            font-weight: 100;
-            font-size: 44px;
-        }
-    </style>
-    {{#if ELECTRON}}
-        <script>
-            var ref = encodeURIComponent(document.referrer);
-            if (window.process && window.process.versions && window.process.versions['electron']) {
-                window.electron = window.process.versions['electron'];
-                window.remote = require('electron').remote;
-                window._$guid$_ = window.remote.getGlobal('guid');
-                window.app_version = window.remote.getGlobal('version');
-                ref = 'lw-' + window.process.platform + '-' + window.app_version + '-' + window.remote.getGlobal('guid');
-                window.onload = function(e){
-                    var content = document.getElementById("content");
-                    if (content) content.className = "electron " + window.process.platform;
-                }
-            }
-        </script>
-    {{/if}}
-</head>
-<body>
-    <main id="content">
-        <h1 class="centerDiv" style="">Loading...</h1>
-    </main>
-</body>
-</html>
+<!doctype html>
+<html>
+
+<head>
+    {{#if INCLUDE_BASE}}
+    <base href="/">
+    {{/if}}
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width" />
+    <meta name="format-detection" content="telephone=no">
+    <title>{{title}}</title>
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' blob:; worker-src 'self' blob:; script-src https://cdnjs.cloudflare.com/ajax/libs/zxcvbn/ 'self' 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; connect-src 'self' *; font-src 'self' data: https://at.alicdn.com; img-src 'self' data:;">
+    <link rel="icon" type="image/x-icon" href="favicon.ico">
+    <style>
+        body {
+            background-color: #2a2a2a;
+        }
+
+        .centerDiv {
+            width: 140px;
+            border-radius: 5px;
+            color: lightgrey;
+            padding: 10px;
+            height: 50px;
+            position: absolute;
+            margin-top: -25px;
+            margin-left: -70px;
+            top: 50%;
+            left: 50%;
+            font-family: "HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+            font-weight: 100;
+            font-size: 44px;
+        }
+
+        .outdated-browser {

```

Comment:
please export the additional css in a separate file and load it only when necessary in the `renderOutdatedWarning` method

Reply:
@sschiessl-bcp done